command/helper/helper.go

```diff
@@ -219,6 +220,20 @@ func ResolveAddr(address string) (*net.TCPAddr, error) {
 	return addr, nil
 }
 
+func ResolveAddrDefaultBindAllInterfaces(address string) (*net.TCPAddr, error) {
+	addr, err := net.ResolveTCPAddr("tcp", address)
+
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse addr '%s': %w", address, err)
+	}
+
+	if addr.IP == nil {
+		addr.IP = net.ParseIP("0.0.0.0")
+	}
+
+	return addr, nil
+}
```

Comment:
Is there any chance we can reuse `ResolveAddr` (call it) here?

Reply:
The only part that's different is the IP setting

Reply:
Maybe change `ResolveAddr` to include a default IP string as a param, and drop this method

Reply:
Removed this func and modified ```ResolveAddr``` in commit b7595a3bfa0d5b769834a4d821b8a19cfc57a9b6

Reply:
Agree, something like
```
func ResolveAddr(address string, defaultIPAddress string) (*net.TCPAddr, error) {
	addr, err := net.ResolveTCPAddr("tcp", address)

	if err != nil {
		return nil, fmt.Errorf("failed to parse addr '%s': %w", address, err)
	}

	if addr.IP == nil {
		addr.IP = net.ParseIP(defaultIPAddress)
	}

	return addr, nil
}
```